README.md

```diff
@@ -1,3 +1,7 @@
+<h1 align="center">
+<img src="docs/_static/logo.png" width="200">
+</h1><br>
```

Comment:
This big logo looks a bit out of place here, I find. I also don't find logos in readme files regularly. Mabye we put such a larger logo into the docs entry page instead?

Reply:
We can put a smaller one. I saw one on numpy github so thought to put it. Easy to copy for people when necessary this way too.

Reply:
I think this is good now.